### PR TITLE
Cybersixgill actionable alerts multi tenancy support

### PR DESCRIPTION
**Pull request by syed-loginsoft**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by yaakovpraisler — CHANGES_REQUESTED** [review]
Hi @syed-loginsoft - Thank you for your contribution!

1. Please sign in the [cla-assistant](https://cla-assistant.io/demisto/content?pullRequest=24579) and fill the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form before we continue.
2. Need to bump the version of the pack and add [release notes](https://xsoar.pan.dev/docs/documentation/release-notes)
3. Please fix the build [validation errors](https://app.circleci.com/pipelines/github/demisto/content/234862/workflows/5f678b3b-2ded-440c-9470-393a5d1afcc0/jobs/660831) and the failed [unit test](https://app.circleci.com/pipelines/github/demisto/content/234862/workflows/5f678b3b-2ded-440c-9470-393a5d1afcc0/jobs/660830) so we can merge.

**Comment by yaakovpraisler** [comment]
Hi @syed-loginsoft - Haven’t heard from you since I published my review.
Do you need any help with something?
Please don’t hesitate to reach out to me here or on Slack.
Thanks again.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py [hunk 1]
```diff
@@ -198,7 +230,8 @@ def fetch_incidents():
 
     filter_alerts_kwargs = get_incident_init_params()
     items = sixgill_alerts_client.get_actionable_alerts_bulk(limit=max_incidents_to_return, from_date=last_fetch_time,
-                                                             sort_order='asc', **filter_alerts_kwargs)
+                                                             sort_order='asc', **filter_alerts_kwargs,
+                                                             organization_id=demisto.params().get('org_id', None))
```
Comment (yaakovpraisler) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py:234]: I can't find such an argument `org_id` in the configuration parameters in the yml file
Please add it there as well

Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py:234]: Added. Please check

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml [hunk 2]
```diff
@@ -16,6 +16,11 @@ configuration:
   required: true
   type: 4
   section: Connect
+- display: Cybersixgill Organization ID
+  name: org_id
+  type: 0
+  required: true
```
[the remark this answers is not in the document]
Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml]: It is now marked as optional and provided a default value.

**Pull request merged**